lxd/scriptlet/starlark_test.go

```diff
@@ -0,0 +1,189 @@
+package scriptlet_test
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/lxc/lxd/lxd/scriptlet"
+	"go.starlark.net/starlark"
```

Comment:
Something isn't correct with this import.

Also please can you separate the imports into separate blocks (separated by newline) in the following order:

- Standard lib imports
- External imports
- Internal imports

Reply:
Ah this error message seemed like a bug. Thanks for the advice, I'll re-order the imports.